CorsixTH/Lua/languages/czech.lua

```diff
@@ -977,32 +992,32 @@ trophy_room = {
   },
 }
 build_room_window = {
-  pick_room_type = "Vyberte Typ Místnosti",
-  pick_department = "Vyberte Oddělení",
+  pick_room_type = "Vyberte typ místnosti",
+  pick_department = "Vyberte oddělení",
   cost = "Náklady: ",
 }
 object = {
-  dna_fixer = "Spravovač DNA",
+  dna_fixer = "Genex 2000",
```

Comment:
As discussed happy with this name as indicated was part of the original translation.
Change the line to have a comment though at the end ( ``-- Your comment here``)

(maybe put it just as the first time this string gets used)